extensions/asde-deployment/package.nls.json

```diff
@@ -35,6 +35,12 @@
 	"vm_password_confirm": "Confirm VM admin password",
 	"dacpac_path": "Dacpac zip file",
 	"azure-info-section-title": "Azure information",
-	"sqlserver-info-section-title": "SQL Server information",
-	"dacpac-zip-files": "Dacpac zip files"
+	"sqlserver-info-section-title": "Azure SQL Edge information",
+	"dacpac-zip-files": "Dacpac zip files",
+	"sql-edge-azure-single-device-display-name": "Existing device of an Azure IoT Hub",
+	"sql-edge-azure-single-device-title": "Deploy Azure SQL Edge to an existing device",
+	"iot-hub-name": "IoT Hub name",
+	"device-id": "Device Id",
```

Comment:
d in "ID" should be capitalized when it's getting displayed in UI/string

Reply:
@Benjin Thanks for the careful review.